packages/runtime/src/next/client.ts

```diff
@@ -129,6 +130,21 @@ export type Sitemap = {
   }[]
 }[]
 
+const getPageAPISchema = z
+  .object({
+    pathname: z.string(),
+    locale: z.string(),
+    alternate: z.array(
+      z.object({
+        pathname: z.string(),
+        locale: z.string(),
+      }),
+    ),
+  })
+  .nullable()
```

Comment:
The result will never be null if the status code is 200

Reply:
True. Updated.